src/tools/cgconfig.c

```diff
@@ -1,18 +1,10 @@
-
-/*
+// SPDX-License-Identifier: LGPL-2.1-only
+/**
```

Comment:
(This is a note to myself as much as anything.)

Having the copyright header be a doxygen comment added a bunch of noise to libseccomp's doxygen output.
https://github.com/seccomp/libseccomp/pull/366/commits/f2548f2fb21d8e2274c372eb48deb6a6f9f134d8#r785130185

Leave it as you have it, but we may want to look more closely at our doxygen sometime.

Reply:
@drakenclimber Thanks for the review.  Interestingly, Doxygen doesn't recognize Copyright. Agree, we might need to audit it.